rusk-wallet/src/lib/prompt.rs

```diff
@@ -367,9 +441,10 @@ pub(crate) fn request_rcvr_addr() -> String {
 
 /// Utility function to check if an address is valid
 fn is_valid_addr(addr: &str) -> bool {
-    bs58::decode(addr).into_vec().is_ok()
+    bs58::decode(addr).into_vec().is_ok() && !addr.is_empty()
```

Comment:
Not sure what's the behaviour of `::decode` if an empty `str` is passed.
Anyway changing the order of this logical operation should result in a faster and safer implementation

Reply:
Indeed, will apply change in follow-up PR 👍